apps/core/scenes/bookingDetail/tripDetails/TripInfoReturn.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { createFragmentContainer, graphql } from '@kiwicom/margarita-relay';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+
+import FromTo from '../../../components/fromTo/FromTo';
+import FromToWrapper from '../../../components/fromTo/FromToWrapper';
+import TripDates from './TripDates';
+import type { TripInfoReturn as BookingType } from './__generated__/TripInfoReturn.graphql';
+
+type Props = {|
+  +data: ?BookingType,
+|};
+
+function TripInfoReturn(props: Props) {
+  return (
+    <>
+      <FromToWrapper>
+        <FromTo
+          textType="attention"
+          fontSize="large"
+          iconColor={defaultTokens.colorIconSecondary}
+          data={props.data}
+          withFlags={true}
+        />
+      </FromToWrapper>
+      <Text style={styles.title} type="secondary">
+        DEPARTURE
```

Comment:
Maybe we should add some uppercase prop to `Text` component, but it's not very important.

Reply:
Maybe, but we can also handle that with styles 🤷‍♂️

Reply:
Yes, it should be also possible, but I'm not sure if it's not buggy (last comment here: https://stackoverflow.com/questions/35813156/how-to-set-text-text-to-upper-case-in-react-native)

Reply:
Btw I noticed that we have prop for `uppercase` in `universal-components` (implemented by styles)